### DISCUSSION TITLE
🎒Approved Student Developer Pack Application FAQs for the Developer Pack is not ready

### DISCUSSION THREAD

**Discussion by Milan-960** [community/community#145477] [GitHub Education] [answered]
### Hi everyone,

My developer Pake is approved and it's been over 2 days but I have not received any emails or anything from the GitHub. and what I found on their common question [here](https://github.com/orgs/community/discussions/111352#user-content-how-long-after-ive-been-approved-will-i-receive-my-academic-benefits) that it will get available soon if not you can ask in the community if anybody knows can you please provide me an update? 

<img width="322" alt="Screenshot 2024-11-24 at 13 38 45" src="https://github.com/user-attachments/assets/fb6b8e8c-7444-4928-9ae0-6ccb6df013aa">

**Comment by davevad93** [comment, marked as answer]
Hi @Milan-960 ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Milan-960 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Milan-960** [reply]
Thank you! @davevad93 will be waiting for an update and to turn it purple! :)

**Comment by theekshana666** [comment]
My developer Pake is approved and it's been over 2 days but I have not received any emails or anything from the GitHub.
